Fix tests for the case of in-place font files.
--- a/test/test.hs
+++ b/test/test.hs
@@ -16,6 +16,6 @@
   when (frontendName == "sdl") $ do
     -- The hacky log priority 0 tells SDL frontend to init and quit at once,
     -- for testing on CIs without graphics access.
-    let args2 = words "--dbgMsgSer --logPriority 0 --newGame 3 --maxFps 100000 --benchmark --stopAfterFrames 50 --automateAll --keepAutomated --gameMode battle --setDungeonRng 7 --setMainRng 7"
+    let args2 = words "--dbgMsgSer --logPriority 0 --newGame 3 --maxFps 100000 --benchmark --stopAfterFrames 50 --automateAll --keepAutomated --gameMode battle --setDungeonRng 7 --setMainRng 7 --fontDir \"GameDefinition/fonts\""
     serverOptions2 <- handleParseResult $ execParserPure defaultPrefs serverOptionsPI args2
     tieKnot serverOptions2
